### DISCUSSION TITLE
My student developer pack is accepted, but the benefits is taking too long; Concern or wait?!

### DISCUSSION THREAD

**Discussion by AydinArif** [community/community#129270] [GitHub Education]
Hey everyone,

I have a question regarding the Student Developer Pack. I submitted my application, and it was accepted on June 14th. However, it has been a week now, and the status still indicates that the benefits are on their way. The notification mentioned that it typically takes three days or more if there is a high volume of requests.

Given this situation, should I wait a bit longer, or would it be advisable to take further action? I would appreciate any guidance or recommendations on the best course of action at this time.

Thank you in advance!
![Screenshot 2024-06-21 004206](https://github.com/community/community/assets/153858317/dd46b65b-ca68-45fb-a9a4-958b8e372c35)

**Comment by mdjeghbala** [comment]
If you can't make a new request, i think you have to wait a bit longer. 


![Screenshot_1](https://github.com/community/community/assets/115470403/71d3ec9b-81e5-4059-883f-7cb84f49e6e3)

You can see this in this link: https://support.github.com/request/education

**Reply by AydinArif** [reply]
Hey,

I just got it today. Thanks for your answer!
